uvtools/dspec.py

```diff
@@ -2062,7 +2066,7 @@ def dpss_operator(x, filter_centers, filter_half_widths, cache=None, eigenval_cu
                                  label='dpss_operator', crit_val=tuple(crit_provided_value[0]))
     if not opkey in cache:
         #check that xs are equally spaced.
-        if not np.all(np.isclose(np.diff(x), np.mean(np.diff(x)))):
+        if not np.all(np.isclose(np.diff(x), np.mean(np.diff(x)), rtol=0., atol=xtol * np.mean(np.diff(x)))):
```

Comment:
There should probably be an absolute value in atol to handle the case where `x` is decreasing.

Reply:
good point. I've adopted this.